CHANGELOG.md

```diff
@@ -231,6 +231,12 @@ The operator now joins using a Kubernetes ServiceAccount token. To validate the
 token, the Teleport Auth Service must have access to the `TokenReview` API.
 The chart configures this for you since v12, unless you disabled `rbac` creation.
 
+The chart now sets a termination grace period of 210 seconds on proxy pods. This
+significantly slows down proxy rollouts but reduces disruption for ongoing SSH
+sessions. If you are setting the `--wait`, `--atomic` Helm flags or are creating
+Helm releases via Terraform, you must increase Helm's timeout. A rule of thumb
```

Comment:
We probably need to refer argo as well https://argo-cd.readthedocs.io/en/stable/user-guide/helm/

In this case, helm is used to inflate the YAML files and then they are applied with kubectl apply.
This might be a problem for rolling out proxies with argo (A lot of users use it do manage teleport clusters)